osmaxx-py/osmaxx/utilities/shortcuts.py

```diff
@@ -32,3 +32,16 @@ def get_list_or_none(klass, *args, **kwargs):
     if not object_list:
         return None
     return object_list
+
+
+def get_actual(obj):
+    """Expands `obj` to the actual object type.
+    """
```

Comment:
No, but the comment should actually be expanded to more than one line explaining what this function does.